src/scrapers/mizrahi.js

```diff
@@ -104,16 +104,18 @@ class MizrahiScraper extends BaseScraperWithBrowser {
       loginUrl: `${LOGIN_URL}`,
       fields: createLoginFields(credentials),
       submitButtonSelector: '#ctl00_PlaceHolderLogin_ctl00_Enter',
-      postAction: async () => waitForNavigation(this.page, { waitUntil: 'networkidle0' }),
+      postAction: async () => {
+        this.request = await this.page.waitForRequest(TRANSACTIONS_REQUEST_URL);
+        waitForNavigation(this.page, { waitUntil: 'networkidle2' });
+      },
       possibleResults: getPossibleLoginResults(),
     };
   }
 
   async fetchData() {
     await this.navigateTo(OSH_PAGE, this.page);
```

Comment:
I tested it and now I remember why it is not in the `postAction`. Do you see the `await this.navigateTo(OSH_PAGE, this.page);`?
The `OSH_PAGE` triggers the request to `TRANSACTIONS_REQUEST_URL`. In Mizrahi, you can select your default page for post login. If you select the "הודעות מהבנקאי" page, the `TRANSACTIONS_REQUEST_URL` will not triggered until you `navigateTo(OSH_PAGE)`,

Reply:
@eshaham @esakal I think I don't approve this PR.

@SafiX Did you have a problem that you opened this PR for?
Because as far I remember, the `waitForRequest` will take the request even if it recorded before the `waitForRequest` command.

Reply:
@baruchiro - yes, with my account which I have the default "default page for post login" the scraper simply doesn't work. The test for it fails as well.

From what I  understand, my PR doesn't work with your configuration?

I guess the solution should have both of our bank ''default page for post login" configuration work.

I'll push a fix to the branch which will handle both cases.

Reply:
@baruchiro - Check out my latest commit to this branch, it should work for both our cases now.

Reply:
@baruchiro - 
ok just FYI, I went to - "הגדרת עמוד כניסה"
And saw that mine was - " פורטל עו''ש "

When I changed it to "תיבת ההודעות" the scraper was working again.

So I think that some people don't change their default post login page, I tested and saw that now, with the additional fix, it will work with both cases.

Would you be able to test as well on your side?